cvat/apps/documentation/user_guide.md

```diff
@@ -252,81 +467,81 @@ Usage examples:
-
-- ``Context menu`` — available on right mouse button.
+- _Manual_ which is created by a person
+- _Semi-automatic_ which is created mainly automatically, but the user provides some data (e.g. interpolation)
+- _Automatic_ which is created automatically without a person in the loop
 
 ---
 ### Workspace — Context menu
 
 Context menu opens by right mouse click.
 
-By clicking inside bounding box the next is available:
-- ``Copy Object URL`` — copying in buffer address of an object on the frame in the task
-- ``Change Color``
-- ``Remove Shape``
-- ``Switch Occluded``
-- ``Switch Lock``
+The next options are available clicking inside bounding box:
+- ``Copy Object URL`` — copying to the buffer address of an object on the frame
+in the task
+- ``Change Color`` — change color of active shape
+- ``Remove Shape`` — deleting the shape
+- ``Switch Occluded`` — attribute is used if an object is occluded by another
+object or isn't fully visible on the frame. Use the ``Q`` shortcut to set the
+property quickly.
+- ``Switch Lock`` — block editing the active shape
 - ``Enable Dragging`` — (only for polygons) allows to adjust polygons position
+- ``Split`` — (only for interpolation bounding box) allows to split an
+interpolated track into two separate tracks. This function is the opposite
+of the merge function.
 
-![](static/documentation/images/image089.jpg) ![](static/documentation/images/image090.jpg)
+![](static/documentation/images/image089.jpg)
+![](static/documentation/images/image090.jpg)
+![](static/documentation/images/image103.jpg)
 
-By clicking on the points of poly-shapes ``Remove`` option is available.
+Clicking on the points of poly-shapes, ``Remove`` option is available.
 
 ![](static/documentation/images/image092.jpg)
 
-By clicking outside any shapes you can either copy ``Frame URL`` (link to present frame) or ``Job URL`` (link from address bar)
+Clicking outside any shapes, you can copy ``Frame URL`` (link to present frame) or ``Job URL`` (link from address bar)
 
 ![](static/documentation/images/image091.jpg)
 
+Applying ``split`` into an interpolated track divides it into two separate tracks.
+
+![](static/documentation/images/gif010.gif)
+
 ---
 ### Settings
 
-Click ``F2`` to access settings menu.
+Press ``F2`` to access settings menu.
 
 ![](static/documentation/images/image067.jpg)
 
 There is ``Player Settings`` which adjusts ``Workspace`` and ``Other Settings``.
 
 In ``Player Settings`` you can:
- - Control step of ``C`` and ``V`` shortcuts
- - Control speed of ``Space``/Play button
- - Show every image in full or zoomed out like previous (reset by default)
-
-  - Enable ``Grid`` when you don't need too small annotation. ``Grid`` can have different opacity, color and cells' size — use ``F2`` to tune settings.
-
+-   Control step of ``C`` and ``V`` shortcuts
+-   Control speed of ``Space``/Play button
+-   Show every image in full or zoomed out like previous (reset by default)
```

Comment:
reset by default - I believe the statement isn't correct. It is enabled by default for interpolation mode and disabled for annotation mode.